src/components/TableGames/TableGames.jsx

```diff
@@ -12,7 +13,12 @@ export const TableGames = ({ ...props }) => {
   const { status, data } = useGames();
 
   if (status === "error") return <div>Oops, something went wrong 👻</div>;
-  if (status === "loading") return <div>loading...</div>;
+  if (status === "loading")
+    return (
+      <div>
+        <Spinner size="5rem" />
```

Comment:
Wieso wrapst du den Spinner in ein div?

Reply:
Äh gute Frage ich glaube weil du davor das hattest und in dem ein h1. Ich änder es!